Test-№1.2-from-09.11.2022/task2.c

```diff
@@ -0,0 +1,70 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <time.h>
+
+void printArray(int *array, int lenOfArray)
+{
+    for (int i = 0; i < lenOfArray; i++)
+    {
+        printf("%d ", array[i]);
+    }
+    printf("\n");
+}
+
+void swap(int *firstElement, int *secondElement)
+{
+    int temp = *firstElement;
+    *firstElement = *secondElement;
+    *secondElement = temp;
+}
+
+bool isCorrectedArray(int *array, int lenOfArray)

```

Comment:
```suggestion
bool isCorrectArray(int *array, int lenOfArray)

```
Корректный массив. А у тебя получился "исправленный массив".